homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -60,295 +55,75 @@ class BMWSensorEntityDescription(SensorEntityDescription):
     "charging_status": BMWSensorEntityDescription(
         key="charging_status",
```

Comment:
Should this be `"charging_level_hv"` instead?

Reply:
Nevermind I see we use that key below. What does the charging status value show? Is that an enum?

Reply:
Yes, it is an additional info to the binary sensor for charging. https://github.com/bimmerconnected/bimmer_connected/blob/dev/bimmer_connected/vehicle_status.py#L38-L47